src/components/DigitalCredentials/dialogs/CredentialNotReceivedDialog.vue

```diff
@@ -0,0 +1,131 @@
+<template>
+  <v-dialog
+    id="credential-not-received-dialog"
+    v-model="dialog"
+    width="45rem"
+    persistent
+    :attach="attach"
+    content-class="credential-not-received-dialog"
+  >
+    <v-card class="pa-5">
+      <v-card-text>
+        <p
+          id="dialog-title"
+          class="warning-title"
+        >
+          Didn't receive a credential offer?
+        </p>
+        <p
+          id="dialog-text"
+          class="warning-text"
+        >
+          Credential offers can take up to a minute, depending on your internet connection. Try the following:
+          <ul>
```

Comment:
Prob need some whitespace between the text and unordered list.

![image](https://github.com/bcgov/business-filings-ui/assets/10002889/aac0323e-a059-40bd-b83b-5d2782925cd6)

The spacing between the list items looks OK but I haven't seen your UI design.

Reply:
I think this is what's in the design. I confer with Kim/Hannah to see if we want to add some spacing

Reply:
Oops, I just noticed a typo above (intenet).

Reply:
Typos are fixed. Resolving and will leave it to Kim/Hannah to review.